packages/kbn-crypto/BUILD.bazel

```diff
@@ -2,7 +2,7 @@
 load("@npm//@bazel/typescript:index.bzl", "ts_config", "ts_project")
 load("@build_bazel_rules_nodejs//:index.bzl", "js_library", "pkg_npm")
 
-PKG_BASE_NAME = "kbn-cypto"
+PKG_BASE_NAME = "kbn-crypto"
```

Comment:
this seems to be a typo (?) unrelated to this PR.

Reply:
Good catch!  I opened https://github.com/elastic/kibana/pull/99935 to address this.